src/mbgl/text/collision_index.cpp

```diff
@@ -56,7 +56,7 @@ float CollisionIndex::approximateTileDistance(const TileDistance& tileDistance,
 }
 
 bool CollisionIndex::isOffscreen(const CollisionBox& box) const {
-    return box.px2 < viewportPadding || box.px1 >= screenRightBoundary || box.py2 < viewportPadding || box.py1 >= screenBottomBoundary;
+    return box.px2 < -viewportPadding || box.px1 >= screenRightBoundary || box.py2 < -viewportPadding || box.py1 >= screenBottomBoundary;
```

Comment:
Can you explain this change? By `box.px2 < viewportPadding` I meant to express "right edge of box is left of left edge of the screen". The projected coordinates come from `projectAndGetPerspectiveRatio`, where the "left edge of the screen" (aka `((-1 + 1) / 2) * width + viewportPadding`) would be equal to `viewportPadding`.

Reply:
Sorry, I misunderstood something. Thanks for catching this.

Reply:
I didn't notice it in the first round, lucky thing we're back-porting the back-port. ;)

The padded coordinate system `CollisionIndex` uses is not super-intuitive, I wonder if we could express it better somehow...